Wikipedia/Code/MWKSavedPageList.m

```diff
@@ -205,7 +205,7 @@ - (void)removeEntriesWithURLs:(NSArray<NSURL *> *)urls {
 }
 
 + (NSArray<NSDictionary *> *)savedEntryDataFromListWithUnknownSchema:(NSDictionary *)data {
-    return [data[MWKSavedPageExportedEntriesKey] wmf_reverseArray];
```

Comment:
removed `wmf_reverseArray` because it's not used elsewhere and these migrations will be removed soon anyway